### PR TITLE
Update kiwi-news-attestate.yaml

### PR DESCRIPTION
**Pull request by TimDaub**
- most of the entries were wrong so I fixed them

### CODE REVIEW

File: data/projects/k/kiwi-news-attestate.yaml [hunk 1]
```diff
@@ -6,26 +6,28 @@ github:
   - url: https://github.com/attestate/awesome-kiwinews
   - url: https://github.com/attestate/delegator2
 blockchain:
-  - address: "0xee324c588ceF1BF1c1360883E4318834af66366d"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:9]: This is the address listed as the payout in the [RPGF3 application](https://vote.optimism.io/retropgf/3/application/0x50565b1e518ab91b14ef50fdab91a63623fb5951e96c18918e36be5aeb6fd297), no?

File: data/projects/k/kiwi-news-attestate.yaml [hunk 2]
```diff
@@ -6,26 +6,28 @@ github:
...
+  - address: "0xebb15487787cbf8ae2ffe1a6cca5a50e63003786"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:9]: ✔️  ERC721DropProxy

File: data/projects/k/kiwi-news-attestate.yaml [hunk 3]
```diff
@@ -6,26 +6,28 @@ github:
...
     tags:
-      - wallet
-      - eoa
+      - contract
     networks:
-      - optimism
-  - address: "0x005241438cAF3eaCb05bB6543151f7AF894C5B58"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:15]: ✔️ Believe this was the address associated with your work as ETH Magician

File: data/projects/k/kiwi-news-attestate.yaml [hunk 4]
```diff
@@ -6,26 +6,28 @@ github:
...
     networks:
-      - mainnet
       - optimism
-  - address: "0x08b7ecfac2c5754abafb789c84f8fa37c9f088b0"
+  - address: "0xea3b341b1f189f8e56b00c8e387b770acae121cf"
     tags:
       - contract
     networks:
       - optimism
-  - address: "0xebb15487787cbf8ae2ffe1a6cca5a50e63003786"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:27]: ✔️ updated to a mainnet contract

File: data/projects/k/kiwi-news-attestate.yaml [hunk 5]
```diff
@@ -6,26 +6,28 @@ github:
...
+  - address: "0x66747bdc903d17c586fa09ee5d6b54cc85bbea45"
     tags:
-      - wallet
-      - eoa
+      - contract
     networks:
-      - mainnet
       - optimism
-  - address: "0x08b7ecfac2c5754abafb789c84f8fa37c9f088b0"
+  - address: "0xea3b341b1f189f8e56b00c8e387b770acae121cf"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:19]: ✔️ PurchaseDelegator

File: data/projects/k/kiwi-news-attestate.yaml [hunk 6]
```diff
@@ -6,26 +6,28 @@ github:
...
+  - address: "0xebb15487787cbf8ae2ffe1a6cca5a50e63003786"
     tags:
-      - wallet
-      - eoa
+      - contract
     networks:
-      - optimism
-  - address: "0x005241438cAF3eaCb05bB6543151f7AF894C5B58"
+      - mainnet
+  - address: "0x66747bdc903d17c586fa09ee5d6b54cc85bbea45"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:14]: ✔️ Kiwi Pass

File: data/projects/k/kiwi-news-attestate.yaml [hunk 7]
```diff
@@ -6,26 +6,28 @@ github:
...
     tags:
-      - wallet
-      - eoa
+      - contract
     networks:
-      - mainnet
       - optimism
-  - address: "0x08b7ecfac2c5754abafb789c84f8fa37c9f088b0"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:22]: ✔️ Included in update as well (different case)

File: data/projects/k/kiwi-news-attestate.yaml [hunk 8]
```diff
@@ -6,26 +6,28 @@ github:
...
+  - address: "0xea3b341b1f189f8e56b00c8e387b770acae121cf"
     tags:
       - contract
     networks:
       - optimism
-  - address: "0xebb15487787cbf8ae2ffe1a6cca5a50e63003786"
+  - address: "0x08b7ECFac2c5754ABafb789c84F8fa37c9f088B0"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:24]: ✔️ Included previously (different case)

File: data/projects/k/kiwi-news-attestate.yaml [hunk 9]
```diff
@@ -6,26 +6,28 @@ github:
...
     tags:
-      - eoa
+      - contract
     networks:
       - optimism
+  - address: "0x1633451fF7cD3688c944634b08829326DF316dD7"
```
Comment (ccerv1) [inline data/projects/k/kiwi-news-attestate.yaml:29]: tags: 
- safe
- wallet

(we handle safes differently than other contracts)

**Review by ccerv1 — CHANGES_REQUESTED** [review]
Thanks Tim!

We haven't done analysis on ethereum mainnet contracts yet, that's why they were missing previously. Thank you for adding them 😄 

A few requests before I can merge your PR:
- [ ]  0xee324c588ceF1BF1c1360883E4318834af66366d is the payout address in [your app attestation](https://vote.optimism.io/retropgf/3/application/0x50565b1e518ab91b14ef50fdab91a63623fb5951e96c18918e36be5aeb6fd297), which is why we included it here. Could you remove that edit?
- [ ]  Could you update the tags for 0x1633451fF7cD3688c944634b08829326DF316dD7 to `safe` and `wallet`. (We handle indexing of multisig contracts differently.)

File: data/projects/k/kiwi-news-attestate.yaml (hunk 1 above)
Comment (TimDaub) [reply data/projects/k/kiwi-news-attestate.yaml:9]: yeah but it is my personal address as we don't have a optimism gnosis safe at the moment. But when it comes to tracking activity on OP my address doesn't really matter. What matters are the contracts that users interact with

Comment (TimDaub) [reply data/projects/k/kiwi-news-attestate.yaml:9]: OK anyways I added it back on

**Comment by TimDaub** [comment]
> Thanks Tim!
> 
> We haven't done analysis on ethereum mainnet contracts yet, that's why they were missing previously. Thank you for adding them 😄
> 
> A few requests before I can merge your PR:
> 
>     * [ ]   0xee324c588ceF1BF1c1360883E4318834af66366d is the payout address in [your app attestation](https://vote.optimism.io/retropgf/3/application/0x50565b1e518ab91b14ef50fdab91a63623fb5951e96c18918e36be5aeb6fd297), which is why we included it here. Could you remove that edit?
> 
>     * [ ]   Could you update the tags for 0x1633451fF7cD3688c944634b08829326DF316dD7 to `safe` and `wallet`. (We handle indexing of multisig contracts differently.)

Hey I just added both points and hope the PR is now good to go. Actually I was wondering, we recently had your blog post about opensource observer on our site (we liked it!) https://news.kiwistand.com/stories?index=0x655e6cd3334fb098fb4d6522ab7d3208b2cb48af2bc63cb3e726c885fc81c501ea447343 and it shows some activity from us, but is this base on the new or the old data then?

**Review by ccerv1 — APPROVED** [review]
TY! Looks good! Let's see if the merge runs.

**Review by ccerv1 — APPROVED** [review]

**Comment by ccerv1** [comment]
Thank you @TimDaub . I just merged your latest additions. I also signed up for Kiwi News -- incredible work, we need this! ✨ 

The data on the site will soon reflect these changes. I may re-run my lists and analysis for RetroPGF a final time to catch data updates, but no guarantees.

**Comment by TimDaub** [comment]
> I also signed up for Kiwi News -- incredible work, we need this! ✨

Amazing I'm so happy to see!!

> The data on the site will soon reflect these changes. I may re-run my lists and analysis for RetroPGF a final time to catch data updates, but no guarantees.

<3

**Pull request merged**